packages/core/src/Nodes/Node.ts

```diff
@@ -14,7 +13,7 @@ export class Node {
 
   constructor(
     public readonly description: NodeDescription,
-    public readonly graph: Graph,
+    // public readonly graph: Graph,
```

Comment:
this doesn't seem to be used anywhere - do we need it?

Reply:
I think I use Graph to access the variables and the custom events?